platform/osgi/platform-osgi-configadmin/src/test/java/org/codice/felix/cm/file/ConfigurationContextImplTest.java

```diff
@@ -238,6 +239,7 @@ public void testGetSanitizedData() {
     testProps.put(FELIX_FILENAME, temporaryFile.toURI());
     testProps.put(FELIX_NEW_CONFIG, true);
     testProps.put(SERVICE_FACTORY_PIDLIST, new ArrayList<>());
+    testProps.put(PROPERTY_REVISION, TEST_PID);
```

Comment:
✏️ Might want to use a value that mimics what we would see in production; how about `5` or something along those lines?

Reply:
Fixed.